libvast/src/system/exporter.cpp

```diff
@@ -217,12 +217,12 @@ exporter(exporter_actor::stateful_pointer<exporter_state> self, expression expr,
     = has_low_priority_option(self->state.options)
         ? query_context::priority::low
         : query_context::priority::normal;
+  VAST_DEBUG("spawned exporter with {} pipelines", pipelines.size());
   self->state.pipeline = pipeline_executor{std::move(pipelines)};
   if (auto err = self->state.pipeline.validate(
         pipeline_executor::allow_aggregate_pipelines::no)) {
-    VAST_ERROR("transformer is not allowed to use aggregate transform {}", err);
-    self->quit();
-    return exporter_actor::behavior_type::make_empty_behavior();
+    VAST_WARN("exporter received aggregation pipelines - this may currently "
+              "cause unstable behavior but pipelines will be applied");
   }
```

Comment:
We should quit with an error and return an empty behavior if `vast.export.max-events` is set to a value other than zero (i.e., unlimited) and if a pipeline is used.

If the pipeline validation fails then we should exit, but we should pass `allow_aggregate_pipelines::yes` here instead.

Reply:
Changed validation in https://github.com/tenzir/vast/pull/2877/commits/def4b4c8412852ae76c067b99d6f0a955d363b4b and implemented checks in https://github.com/tenzir/vast/pull/2877/commits/5b1bb55dc3de6e7e43e60ce95996283cb0ea55c2